Revit_Core_Engine/Create/Solid.cs

```diff
@@ -94,5 +94,62 @@ public static Solid Solid(IList<CurveLoop> boundaries, double bottomElevation, d
         }
 
         /***************************************************/
+
+        [Description("Create Solid object from ViewPlan (based on cropbox and view range).")]
+        [Input("view", "ViewPlan to get the solid from.")]
+        [Input("transform", "Transform of the link model.")]
+        [Output("solid", "Solid object created from the ViewPlan.")]
+        public static Solid Solid(this ViewPlan view, Transform transform)
+        {
+            if (view == null || transform == null)
```

Comment:
I do not get it: can't i create the solid from view if it is not in link?

Reply:
Changed it to work in this case.